lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -476,14 +545,50 @@ defmodule ExUnit.DocTestTest do
         doctest ExUnit.DocTestTest.IndentationNotEnough
       end
     end
+  end
+
+  test "fails in fenced indentation mismatch" do
+    assert_raise ExUnit.DocTest.Error,
+      ~r[test/ex_unit/doc_test_test.exs:\d+: indentation level mismatch: " iex> bar = 2", should have been 0 spaces], fn ->
```

Comment:
Please escape the "."  in the new regexes . I am including this change for the existing lines in the another PR I'm working on. thank you.